integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -61,14 +66,27 @@ private static String getInsertWithPartitionSql(
   // Whether supports [CLUSTERED BY col_name3 SORTED BY col_name INTO num_buckets BUCKETS]
   protected abstract boolean supportsSparkSQLClusteredBy();
 
-  // Use a custom database not the original default database because SparkIT couldn't read&write
-  // data to tables in default database. The main reason is default database location is
+  protected abstract boolean supportsPartition();
+
+  // Use a custom database not the original default database because SparkCommonIT couldn't
+  // read&write data to tables in default database. The main reason is default database location is
   // determined by `hive.metastore.warehouse.dir` in hive-site.xml which is local HDFS address
   // not real HDFS address. The location of tables created under default database is like
   // hdfs://localhost:9000/xxx which couldn't read write data from SparkCommonIT. Will use default
   // database after spark connector support Alter database xx set location command.
   @BeforeAll
   void initDefaultDatabase() {
+    // cleanup the metastore_db directory in embedded mode
+    // to avoid the exception about `ERROR XSDB6: Another instance of Derby may have already booted
+    // the database /home/runner/work/gravitino/gravitino/integration-test/metastore_db`
+    File hiveLocalMetaStorePath = new File("metastore_db");
```

Comment:
I couldn't understand why add this code,   is `metastore_db` introduced by this PR?

Reply:
In embedded mode, derby acts as the backend database for the hive metastore and creates a directory named `metastore_db` to store metadata, supporting only one connection at a time. Previously, only `SparkHiveCatalogIT` accessed derby without any exceptions. Now, `SparkIcebergCatalogIT` exists at the same time. This problem will occur when `SparkIcebergCatalogIT` is initialized after the `Sparkhivecatalogit` is executed. The main reason is that the lock file in the `metastore_db` directory is not cleaned so that a new connection cannot be created, so a clean operation is done here to ensure that a new connection can be created.

Reply:
https://github.com/datastrato/gravitino/blob/main/integration-test/build.gradle.kts#L193-L197

There is also a cleanup operation, but this is done after all test cases have been executed, not before SparkIcebergCatalogIT is executed.